modules/flex/versions.tf

```diff
@@ -25,7 +25,8 @@ terraform {
   }
 
   provider_meta "google" {
```

Comment:
Lets update this to goog-beta since that is used by the resource, similarly above for version constraint as well. Also bump min tf version to 1.3

Reply:
Done.